### PR TITLE
Added option for minimal build

### PR DESCRIPTION
**Pull request by skogtake**
Added webpack config for building the project without certain modules: Default UI, Dash, Cast and shaka-player.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
   try {
     var kalturaPlayer = KalturaPlayer.setup(config);
     kalturaPlayer.loadMedia({entryId: '0_wifqaipd'});
+    document.querySelector('.kaltura-player-container').appendChild(kalturaPlayer.getView());
+    kalturaPlayer = kalturaPlayer;
```
Comment (medin-n) [inline src/index.html]: no need for this

File: package.json [hunk 2]
```diff
@@ -21,7 +23,6 @@
     "playkit-dev-ott:start": "playkit-dev start ott",
     "playkit-dev:stop": "playkit-dev stop",
     "playkit-rel": "playkit-rel start",
-    "precommit": "lint-staged",
```
Comment (elsassph) [inline package.json]: Why removing linting?

Comment (medin-n) [reply package.json]: image

Comment (elsassph) [reply package.json]: Mayyybe just fix the errors? :D

Comment (medin-n) [reply package.json]: reverted

File: README.md [hunk 3]
```diff
@@ -1,36 +1,8 @@
-# Kaltura Player JS Platform - Cloud TV and OVP Media Players Based on the [PlayKit JS Player]
+# Massive Fork of Kaltura Player
```
Comment (elsassph) [inline README.md]: Please revert the readme and add an extra readme for the minimal version

Comment (medin-n) [reply README.md]: done

**Comment by elsassph** [comment]
Approved.

**Pull request closed without merging**